utils/utils.go

```diff
@@ -50,3 +53,16 @@ func AddLabel(obj metav1.Object, key string, value string) {
 	}
 	labels[key] = value
 }
+
+// GetCSVByPrefix returns CSV from the list based on the prefix
+func GetCSVByPrefix(csvList opv1a1.ClusterServiceVersionList, name string) *opv1a1.ClusterServiceVersion {
+	var csv *opv1a1.ClusterServiceVersion = nil
+	for index := range csvList.Items {
+		candidate := &csvList.Items[index]
+		if strings.HasPrefix(candidate.Name, name) {
+			csv = candidate
+			break
+		}
+	}
+	return csv
+}
```

Comment:
The `utils.go` was intended for generic utils once specific to a single domain. 
Either move this func to the controller or generalize to 

```func SelectByPrefix(list []metav1.Object, name string) metav1.Object```